rest/data/distro.go

```diff
@@ -4,21 +4,15 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/evergreen-ci/birch"
 	"github.com/evergreen-ci/evergreen"
 	"github.com/evergreen-ci/evergreen/model"
 	"github.com/evergreen-ci/evergreen/model/distro"
 	"github.com/evergreen-ci/evergreen/model/host"
 	"github.com/evergreen-ci/gimlet"
-	"github.com/pkg/errors"
 )
 
-// DBDistroConnector is a struct that implements the Distro related methods
-// from the Connector through interactions with the backing database.
-type DBDistroConnector struct{}
-
 // FindDistroById queries the database to find a given distros.
-func (dc *DBDistroConnector) FindDistroById(distroId string) (*distro.Distro, error) {
+func FindDistroById(distroId string) (*distro.Distro, error) {
```

Comment:
Since we're just wrapping one function, we should just call the FindOne function directly